corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -229,20 +229,29 @@ def __init__(self, report, filter_slug):
         super(LocationChoiceProvider, self).__init__(report, filter_slug)
         self.include_descendants = False
         self.show_full_path = False
+        self.location_type = None
 
     def configure(self, spec):
         self.include_descendants = spec.get('include_descendants', self.include_descendants)
         self.show_full_path = spec.get('show_full_path', self.show_full_path)
+        self.location_type = spec.get('location_type', self.location_type)
 
     def _locations_query(self, query_text, user):
-        active_locations = SQLLocation.active_objects
+        locations = SQLLocation.active_objects
         if query_text:
-            return active_locations.filter_path_by_user_input(
+            locations = locations.filter_path_by_user_input(
                 domain=self.domain,
                 user_input=query_text
-            ).accessible_to_user(self.domain, user)
+            )
+
+        if self.location_type:
+            try:
+                locations = locations.filter(location_type__code=self.location_type)
+            except SQLLocation.DoesNotExist:
```

Comment:
When would this exception be hit? I believe the previous line should be returning a queryset, but I think `DoesNotExist` is only thrown when trying to access an instance of a model that doesn't exist, so I'm not sure what this exception will catch

Reply:
Ha, true, I will remove this